icons/poi_college.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" xmlns="http://www.w3.org/2000/svg">
+ <path d="M 10,4 2.5,7.5 4,8.166 C 4,10.143784 4,13.648377 4,16 H 5.5 V 8.9 L 10,11.000001 17.5,7.5 Z M 14.5,9.9 10,12.000001 C 9.1960909,11.604612 7.5793651,10.855791 6.5186269,10.367479 6.503761,10.408608 6.4919116,14.590274 6.4919116,14.590274 7.4564094,15.070546 8.7760134,15.516238 10,15.5 12.025778,15.47312 14.050277,14.429627 14.5,14 Z" style="stroke:#f9f5f0;stroke-opacity:1;paint-order:stroke fill markers;stroke-width:2;stroke-dasharray:none"/>
```

Comment:
This looks good to me, I'd just tweak it a bit:
* Change stroke color to white
* Height (and realign path due to that change)
* Remove unnecessary styling and cleanup what remains

```suggestion
<svg width="20" height="14" xmlns="http://www.w3.org/2000/svg">
 <path d="m10 1.1-7.5 3.5 1.5 0.666v7.734h1.5v-7l4.5 2.1 7.5-3.5zm4.5 5.9-4.5 2.1c-0.80391-0.39539-2.4206-1.1442-3.4814-1.6325-0.014866 0.041129-0.026715 4.2228-0.026715 4.2228 0.9645 0.48027 2.2841 0.92596 3.5081 0.90973 2.0258-0.02688 4.0503-1.0704 4.5-1.5z" stroke="#fff" stroke-width="2" paint-order="stroke fill"/>
```

Reply:
My read of the [contributing guidelines](https://github.com/ZeLonewolf/openstreetmap-americana/blob/main/CONTRIBUTING.md#color-scheme-1) was that the borders were supposed to be this shade of off-white, I guess to match the background?

Also not sure if the dimensions are supposed to be 20x20 even if the icon doesn't need that much room, or if the dimensions should be such that the icon just fits (or if the svg dimensions even matter).

Reply:
As for dimensions, I believe most of our icons that don't fill the full 20x20 have reduced the SVG size to the fewest pixels necessary (there may be exceptions and we should probably clarify our position on that).

Reply:
Certainly none of the ones I've added have been (e.g. school, police badge, museum), clarification welcome.

Reply:
Marked this as resolved. If there's consensus for it, in the future the icon dimensions could easily be shrunk from 20x20 to just fitting.